### ISSUE TITLE
Allow scrolling with the scroll-wheel while dragging

### ISSUE THREAD

**Issue by inzanez** [isaacHagoel/svelte-dnd-action#487]
Hi

just started playing around with your library, great implementation! I do have one question though: sometimes, scrolling by moving the dragged item to the top / bottom does not work smoothly,...not sure why that is, but anyhow: it would be quite handy if it were possible to allow scrolling using the mouse wheel too?...

**Comment by isaacHagoel** [comment]
I didn't have a mouse with a wheel for ages :)
what happens when you try to use it to scroll?

**Comment by inzanez** [comment]
@isaacHagoel Hehe, ok :-) I just doesn't do anything. Stays in place. So it seems scrolling is disabled for some reason. I thought that was on purpose maybe.

**Comment by isaacHagoel** [comment]
Does it not work (the wheel) only during drag?

**Comment by LiddellHunter** [comment]
I also have the same issue that I was hoping to have addressed. I have the dnd inside of a <div> that is scrollable when many elements are added. I can scroll without dragging but once I drag an element I am no longer able to scroll until I release the element.
Though a top/bottom scroll works. It's just clunky. Top/Bottom meaning dragging an element to the top/bottom of the dnd zone and the browser does the work to scroll.

**Comment by isaacHagoel** [comment]
What is exactly the ask?
An option to disable auto scrolling? Have the two working together (what if
they are in conflict)?

**Comment by LiddellHunter** [comment]
I think what OP was getting at is that when dragging an element in a scrollable container, the user cannot use the scroll wheel to scroll up/down.

**Comment by kkarpeev** [comment]
Maybe you can do it like [here](https://tarb.github.io/svelte-dnd-list/scroll). That's svelte-dnd-list (uses requestAnimationFrame). Or like in plain-draggable (options that specify arrays of 1-3 distances in px or % from the edges to enable 1-3 scroll speeds). Or something like that.
Scrolling is a real pain for me, too.

**Comment by isaacHagoel** [comment]
Is this still an issue? can someone with a scroll wheel check for me? :)
version 0.9.41 and up should deal with scrolling much better

**Comment by tonydangblog** [comment]
I checked and scrolling with the mouse wheel works for me while dragging. It worked when I was on `0.9.38` and still works in latest `0.9.42`. I tested on a Mac with Chrome Version 123.0.6312.87 (Official Build) (arm64).

In addition, in previous version `0.9.38`, when dragging down, the page would not scroll. This has also be resolved in version `0.9.42` for me both on desktop and mobile.

https://github.com/isaacHagoel/svelte-dnd-action/assets/63323230/aaec7897-e11b-4b29-b100-80a72354d326
